libs/local_lcos/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ add_hpx_module(local_lcos
   SOURCES ${local_lcos_sources}
   HEADERS ${local_lcos_headers}
   COMPAT_HEADERS ${local_lcos_compat_headers}
+  EXCLUDE_FROM_GLOBAL_HEADER
+    "hpx/local_lcos/force_linking.hpp"
```

Comment:
Good catch on this one. I wonder if we can just use the generated force linking functionality instead though...

Reply:
Yes, I thought about it, but then decided that this would be outside of the scope of this PR.